### ISSUE TITLE
test_scale_pvc_expand.py timeouts result in pvc not being found

### ISSUE THREAD

**Issue by wusui** [red-hat-storage/ocs-ci#3456]
Information is in attached log file.
[pvc_expand.log](https://github.com/red-hat-storage/ocs-ci/files/5625720/pvc_expand.log)

**Comment by wusui** [comment]
This problem may be similar to other issues seen on scale tests.  I will open a bug report for all of these.

**Comment by wusui** [comment]
This looks like it could be an independent problem from the other timeouts.

**Comment by wusui** [comment]
3455, 3456, and 3457 seem like independent timeout issues although I could be wrong...

**Comment by tiffanyn108** [comment]
The issue is seen when 1500 PVC creation using thread.  Many thread exception is seen:
``` Exception in thread Thread-2906:
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/threading.py", line 932, in _bootstrap_inner
    self.run()
  File "/usr/local/lib/python3.8/threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "/home/wusui/ocs-ci/ocs_ci/ocs/ocp.py", line 694, in wait_for_delete
    describe_out = self.describe(resource_name=resource_name)
  File "/home/wusui/ocs-ci/ocs_ci/ocs/ocp.py", line 271, in describe
    return self.exec_oc_cmd(command, out_yaml_format=False)
  File "/home/wusui/ocs-ci/ocs_ci/ocs/ocp.py", line 144, in exec_oc_cmd
    out = run_cmd(
  File "/home/wusui/ocs-ci/ocs_ci/utility/utils.py", line 451, in run_cmd
    completed_process = exec_cmd(cmd, secrets, timeout, ignore_error, **kwargs)
  File "/home/wusui/ocs-ci/ocs_ci/utility/utils.py", line 505, in exec_cmd
    raise CommandFailed(
ocs_ci.ocs.exceptions.CommandFailed: Error during execution of command: oc -n namespace-test-acbf5acf812b404a98fa9a98c9143711 --kubeconfig /home/wusui/ocs-ci/auth/kubeconfig describe PersistentVolumeClaim pvc-test-81685d95dbfa4b27ac3aa190f41de53e.
Error is Error from server (NotFound): persistentvolumeclaims "pvc-test-81685d95dbfa4b27ac3aa190f41de53e" not found ```

**Comment by ramkiperiy** [comment]
This issue should have been resolved with 4.7 CSI driver improvements.

**Comment by ramkiperiy** [comment]
Closing this issues, since with latest 4.7 CSI enhancements PVCs creations are faster and not hitting this kind of problems.
